extension-types.md

```diff
@@ -39,18 +39,47 @@ extension name or metadata).
 ## Extension metadata
 
 When GeoArrow-encoded Arrays have the `ARROW:extension:metadata` metadata
-field set, it must be serialized as a UTF-8 encoded JSON object. The following
-keys in the JSON metadata object are supported:
-
-- `crs`: A JSON object describing the coordinate reference system (CRS)
-  using [PROJJSON](https://proj.org/specifications/projjson.html).
-  This key can also be omitted if the producer does not have any
-  information about the CRS. Note that regardless of the axis
-  order specified by the CRS, axis order will be interpreted
-  according to the wording in the
+field set, it must be serialized as a UTF-8 encoded JSON object. The extension
+metadata specification is intentionally aligned with the
+[GeoParquet column metadata specifification](https://github.com/opengeospatial/geoparquet/blob/v1.1.0/format-specs/geoparquet.md#metadata).
+The following keys in the JSON metadata object are supported:
+
+- `crs`: One of:
+
+    - A JSON object describing the coordinate reference system (CRS)
+      using [PROJJSON](https://proj.org/specifications/projjson.html).
+    - A string containing a serialized CRS representation. This option
+      is intended as a fallback for producers (e.g., database drivers or
+      file readers) that are provided a CRS in some form but do not have the
+      means to convert it to PROJJSON.
+    - Omitted, indicating that the producer does not have any information about
+      the CRS.
+
+  For maximum compatibility, producers should write PROJJSON where possible.
+  Note that regardless of the axis order specified by the CRS, axis order will be interpreted
   [GeoPackage WKB binary encoding](https://www.geopackage.org/spec130/index.html#gpb_format):
   axis order is always (longitude, latitude) and (easting, northing)
   regardless of the the axis order encoded in the CRS specification.
+
+- `crs_type`: An optional string disambiguating the value of the `crs` field.
+  Must be omitted or a string value of:
+
+  - `"projjson"`: Indicates that the `"crs"` field was written as
```

Comment:
Isn't this redundant? If `crs` is a JSON object, then `crs_type` _must_ be `"projjson"`, right? In that case, maybe better to remove this option. That would also be backwards-compatible with the existing spec, where a `crs` key with a JSON object is always PROJJSON, without needing to check the `crs_type` field.

In that case `crs_type` would only provide information about `crs` when the type of `crs` is a string.